base/cloudbuild.yaml.in

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args: ['build', '--tag=${IMAGE}', './base']
+    id: BUILD
+  - name: gcr.io/gcp-runtimes/structure_test
+    args: ['--image', '${IMAGE}', '-v', '--config', '/workspace/test/test_config.json']
```

Comment:
ah nevermind actually. this is an argument being passed to the structure test binary, which uses a Go library to read in the config file. so it needs to be the fully qualified path.